examples/prometheus/prometheus.yaml

```diff
@@ -48,10 +53,12 @@ objects:
     name: prometheus
     namespace: "${NAMESPACE}"
   spec:
+    host: "${PROMETHEUS_HOSTNAME}"
     to:
       name: prometheus
     tls:
       termination: Reencrypt
+      destinationCACertificate: "${SERVICE_SIGNER_CERT}"
```

Comment:
fixed in https://github.com/openshift/origin/pull/15078.
closing this PR to avoid confusion